File: mvc6-angular/Angle-MVC6-Angular/src/Angle-MVC6-Angular/Controllers/PagesController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using Microsoft.AspNetCore.Mvc;

namespace Angle_MVC6_Angular.Controllers
{
    public class PagesController : Controller
    {
        // GET: Pages
        public IActionResult Error404()
        {
            return View();
        }
        public IActionResult Lock()
        {
            return View();
        }
        public IActionResult Login()
        {
            return View();
        }
        public IActionResult Page()
        {
            return View();
        }
        public IActionResult Recover()
        {
            return View();
        }
        public IActionResult Register()
        {
            return View();
        }
        public IActionResult Maintenance()
        {
            return View();
        }
        public IActionResult Error500()
        {
            return View();
        }
    }
}

File: backend-rails/Angle/app/controllers/pages_controller.rb
class PagesController < ApplicationController
  layout 'pages'

  def login
  end

  def register
  end

  def recover
  end

  def lock
  end

  # set another layout for a specific action
  def template
    render :layout => 'application'
  end

  def notfound
  end

  def error500
  end

  def maintenance
  end
end
